### ISSUE TITLE
JS Stack Trace

### ISSUE THREAD

**Issue by eddiemoore** [davidtsai/node-geoip2#8]
I'm hitting this issue. I think it's an issue with the `.gyp` stuff. Running on node 6.
```shell
(node) v8::ObjectTemplate::Set() with non-primitive values is deprecated
(node) and will stop working in the next major release.

==== JS stack trace =========================================

Security context: 0xffdfdec9fa9 <JS Object>#0#
    1: .node [module.js:568] [pc=0x29d0fa3f4604] (this=0x10a9db3ee9f9 <an Object with map 0x10929e2ae641>#1#,module=0x294c25c7f449 <a Module with map 0x10929e2181b9>#2#,filename=0x294c25c7f421 <String[64]: /XXX/node_modules/geoip2/lib/node_mmdb.node>)
    2: load [module.js:~447] [pc=0x29d0f9b260f6] (this=0x294c25c7f449 <a Module with map 0x10929e2181b9>#2#,filename=0x294c25c7f421 <String[64]: /XXX/node_modules/geoip2/lib/node_mmdb.node>)
    3: tryModuleLoad(aka tryModuleLoad) [module.js:415] [pc=0x29d0f9738edd] (this=0xffdfde04189 <undefined>,module=0x294c25c7f449 <a Module with map 0x10929e2181b9>#2#,filename=0x294c25c7f421 <String[64]: /XXX/node_modules/geoip2/lib/node_mmdb.node>)
    4: _load [module.js:~381] [pc=0x29d0f9f4bbad] (this=0x10a9db3eea79 <JS Function Module (SharedFunctionInfo 0x10a9db328801)>#3#,request=0x3453ce4b
==== C stack trace ===============================

 1: v8::Template::Set(v8::Local<v8::Name>, v8::Local<v8::Data>, v8::PropertyAttribute)
 2: init(v8::Local<v8::Object>)
 3: node::DLOpen(v8::FunctionCallbackInfo<v8::Value> const&)
 4: v8::internal::FunctionCallbackArguments::Call(void (*)(v8::FunctionCallbackInfo<v8::Value> const&))
 5: v8::internal::MaybeHandle<v8::internal::Object> v8::internal::(anonymous namespace)::HandleApiCallHelper<false>(v8::internal::Isolate*, v8::internal::(anonymous namespace)::BuiltinArguments<(v8::internal::BuiltinExtraArguments)1>)
 6: v8::internal::Builtin_HandleApiCall(int, v8::internal::Object**, v8::internal::Isolate*)
 7: 0x29d0f960961b
(node) v8::ObjectTemplate::Set() with non-primitive values is deprecated
(node) and will stop working in the next major release.

==== JS stack trace =========================================

Security context: 0xffdfdec9fa9 <JS Object>#0#
    1: .node [module.js:568] [pc=0x29d0fa3f4604] (this=0x10a9db3ee9f9 <an Object with map 0x10929e2ae641>#1#,module=0x294c25c7f449 <a Module with map 0x10929e2181b9>#2#,filename=0x294c25c7f421 <String[64]: /XXX/node_modules/geoip2/lib/node_mmdb.node>)
    2: load [module.js:~447] [pc=0x29d0f9b260f6] (this=0x294c25c7f449 <a Module with map 0x10929e2181b9>#2#,filename=0x294c25c7f421 <String[64]: /XXX/node_modules/geoip2/lib/node_mmdb.node>)
    3: tryModuleLoad(aka tryModuleLoad) [module.js:415] [pc=0x29d0f9738edd] (this=0xffdfde04189 <undefined>,module=0x294c25c7f449 <a Module with map 0x10929e2181b9>#2#,filename=0x294c25c7f421 <String[64]: /XXX/node_modules/geoip2/lib/node_mmdb.node>)
    4: _load [module.js:~381] [pc=0x29d0f9f4bbad] (this=0x10a9db3eea79 <JS Function Module (SharedFunctionInfo 0x10a9db328801)>#3#,request=0x3453ce4b 1: v8::Template::Set(v8::Local<v8::Name>, v8::Local<v8::Data>, v8::PropertyAttribute)
 2: init(v8::Local<v8::Object>)
 3: node::DLOpen(v8::FunctionCallbackInfo<v8::Value> const&)
 4: v8::internal::FunctionCallbackArguments::Call(void (*)(v8::FunctionCallbackInfo<v8::Value> const&))
 5: v8::internal::MaybeHandle<v8::internal::Object> v8::internal::(anonymous namespace)::HandleApiCallHelper<false>(v8::internal::Isolate*, v8::internal::(anonymous namespace)::BuiltinArguments<(v8::internal::BuiltinExtraArguments)1>)
 6: v8::internal::Builtin_HandleApiCall(int, v8::internal::Object**, v8::internal::Isolate*)
 7: 0x29d0f960961b
```

**Comment by pandrese** [comment]
I can confirm that updating `nan` dependency to version `2.3.3` solves this problem.

**Comment by davidtsai** [comment]
Fixed in 1.0.3
